src/src/WebServer/NotificationPage.cpp

```diff
@@ -191,6 +201,7 @@ void handle_notifications() {
     addSelector_Foot();
 
     addHelpButton(F("EasyNotifications"));
+    addHelpButton(F("RTD_Notifications.html"));
```

Comment:
Probably better to call `addRTDHelpButton(F("Notify/_Notifications.html"));` directly, as we might remove the old Wiki help at some point in the future.

Reply:
I will make the change. I saw that both functions were used in ESPEasy, so I did a coin toss decision. I blame the coin.